cms/djangoapps/contentstore/utils.py

```diff
@@ -94,7 +94,7 @@ def _remove_instructors(course_key):
 
     try:
         remove_all_instructors(course_key)
-    except Exception as err:
+    except Exception as err:  # pylint: disable=too-general-exception
```

Comment:
The quality test was failing because of this line, so I figured I'd disable it for now and make a jira ticket to make a better solution. Does that work for you, @mulby and @MatthewPiatetsky ?